src/west/app/main.py

```diff
@@ -394,6 +397,15 @@ def load_extension_specs(self):
 
             self.extension_groups[path] = filtered
 
+    def load_aliases(self):
+        if not self.config:
+            return
+
+        for item in self.config.items():
+            ids = item[0].split('.', 1)
+            if ids[0] == 'alias':
+                self.aliases[ids[1]] = Alias(ids[1], item[1])
```

Comment:
Codestyle nit: this looks like a good use case for list comprehension:

```python
self.aliases = [ Alias(key.split('.',1)[1], value) for
      key, value in self.config.items() if key.split('.')[0] == 'alias' ]
```

Reply:
The aliases are a dictionary, so I changed it a bit, PTAL.